src/util/constants.js

```diff
@@ -1,6 +1,7 @@
 export const colors = {
-	primary: 'teal.500',
-	secondary: 'teal.50',
-	neutralGray: 'gray.500',
-	neutralWhite: 'white',
-};
\ No newline at end of file
+    primary: 'teal.500',
+    secondary: 'teal.50',
+    neutralGray: 'gray.500',
+    neutralLighterGray: 'gray.200',
+    neutralWhite: 'white',
+};
```

Comment:
add colorScheme property with value 'teal'

Reply:
The color schemes for different components vary, hence why we are using primary and secondary.

Reply:
Great catch!